deploy/sdk/src/dynamo/sdk/cli/serve_standalone.py

```diff
@@ -0,0 +1,149 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import inspect
+import logging
+from typing import Any, Dict, get_type_hints
+
+from pydantic import BaseModel, ConfigDict
+
+from dynamo.runtime import Component, DistributedRuntime, dynamo_endpoint, dynamo_worker
+from dynamo.sdk.cli.utils import configure_target_environment
+from dynamo.sdk.core.protocol.interface import DynamoTransport
+from dynamo.sdk.core.runner import TargetEnum
+
+logger = logging.getLogger(__name__)
+
+# Use Dynamo target (this is the only supported one)
+configure_target_environment(TargetEnum.DYNAMO)
+
+
+class DynamoContext(BaseModel):
+    """Context object for the service that is injected into components that declare it as a typed parameter."""
+
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+
+    runtime: DistributedRuntime
+    component: Component
+    endpoints: Dict[str, Any]
+    name: str
+    namespace: str
+
+
+async def serve(service, *args, **kwargs):
+    # 1. Create a runtime/dyn worker if one is not passed
```

Comment:
This should likely be a documentation comment (aka docustring) and not just a comment.